api/odl2.py

```diff
@@ -200,12 +200,7 @@ def _extract_publication_metadata(self, feed, publication, data_source_name):
                 if parsed_license is not None:
                     licenses.append(parsed_license)
 
-                # DPLA feed doesn't have information about a DRM protection used for audiobooks.
-                # We want to try to extract that information from the License Info Document it's present there.
                 license_formats = set(odl_license.metadata.formats)
-                if parsed_license and parsed_license.content_types:
-                    license_formats |= set(parsed_license.content_types)
```

Comment:
Why can this go away? Was it vestigial/effectively a no-op? Or did I miss this moving somewhere else in another PR?

Reply:
Effectively we used to get `application/audiobook+json; protection=http://www.feedbooks.com/audiobooks/access-restriction` from the license status document, and the other types from the feed, but they are now all available in the feed. And hopefully only available if they are in fact available.

Reply:
🙏🏽